dropwizard-index-page/src/main/java/com/palantir/indexpage/TemplatedAsset.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.indexpage;
+
+import com.google.common.base.Optional;
+import java.util.Map;
+
+/**
+ * Simple interface used by both {@link ClasspathIndexPage} and {@link FileSystemIndexPage} to return the resolved
+ * asset template.
+ */
+public interface TemplatedAsset {
```

Comment:
name on this feels off. shouldn't it be an `IndexPage`, considering this is an `index-page` bundle?